packages/httpsnippet-client-api/src/index.ts

```diff
@@ -151,20 +193,22 @@ const client: Client<APIOptions> = {
       );
     }
 
-    let sdkPackageName;
-    let sdkVariable: string;
-    if (opts.api.identifier) {
-      sdkPackageName = opts.api.identifier;
+    let sdkPackageName: string;
+    if (opts.api.packageName) {
+      sdkPackageName = opts.api.packageName;
+    } else {
+      const registryUUID = getProjectPrefixFromRegistryUUID(opts.api.registryURI);
+      sdkPackageName = registryUUID || 'company-name';
+    }
 
-      sdkVariable = camelCase(opts.api.identifier);
+    let sdkVariable: string = 'sdk';
```

Comment:
i feel like the SDK variable should fall back to the `sdkPackageName` if `variableName` isn't present as opposed to `sdk`, thoughts?

```suggestion
    let sdkVariable = sdkPackageName;
```

Reply:
I thought that we wanted this custom variable name stuff to be completely custom for folks who want it but I'm ok with this.

Reply:
agreed, i like the idea of a custom variable name! this is just in the fallback case